libs/ui-lib/lib/ocm/components/hosts/HostsDiscoveryTable.tsx

```diff
@@ -55,6 +59,51 @@ const HostsDiscoveryTable = ({ cluster }: HostsDiscoveryTableProps) => {
     onMassDeleteHost,
     ...modalProps
   } = useHostsTable(cluster);
+
```

Comment:
I'd extract everything you added to this file to a separate hook
for example `useLateBinding(..)`

which would also return error/loading state so that we can show the binding error to the users/disable next while in progress.